editstyle.css

```diff
@@ -0,0 +1,111 @@
+*{
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+.container {
+    max-width: 1300px;
+    margin: 0 auto;
+    padding: 20px;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
```

Comment:
Note: This is just a suggestion you may keep this style for now.
Because containers are thought of as global styles, avoid applying min-height directly to them. Instead, pass another class and apply the min height style.